operator/controllers/semconv/keptn.go

```diff
@@ -1,26 +1,19 @@
 package semconv
```

Comment:
I would move this pkg into `v1alpha1` and move the current keys inside `commont` back here. 
This way, it is easier to understand the purpose

Reply:
ok